engine-rest/engine-rest-openapi/src/main/templates/paths/user/options.ftl

```diff
@@ -20,14 +20,26 @@
         last = true
         desc = "Request successful."
         examples = ['"example-1": {
-                       "summary": "GET `/user/aUserId`",
+                       "summary": "GET `/user`",
                        "value": {
-                         "links":[
-                          {"method":"GET","href":"http://localhost:8080/camunda/api/engine/engine/default/user/peter/profile","rel":"self"},
-                          {"method":"DELETE","href":"http://localhost:8080/camunda/api/engine/engine/default/user/peter","rel":"delete"},
-                          {"method":"PUT","href":"http://localhost:8080/camunda/api/engine/engine/default/user/peter/profile","rel":"update"}
+                          "links": [
```

Comment:
Let's make this changes in the docs-manual as well.

Reply:
Ok, I'll add a second example.